user32.go

```diff
@@ -1854,6 +1854,7 @@ var (
 	releaseCapture              *windows.LazyProc
 	releaseDC                   *windows.LazyProc
 	removeMenu                  *windows.LazyProc
+	deleteMenu                  *windows.LazyProc
```

Comment:
Please keep functions sorted.

Reply:
Wow, that's quite some stuff. Thanks.

Reply:
Sorted them and force-pushed. Thanks for your feedback.